docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -51,7 +92,7 @@ for Hive. In the following sections, Hive is running on Hadoop MapReduce in this
 
 > Tips：All the following Hive CLI examples are also applicable to Hive Beeline. You can try these commands out in Beeline shell.
 
-> Tips：All the following examples are also applicable to Alluxio in fault tolerant mode with Zookeeper. 
+> Tips：All the following examples are also applicable to Alluxio in fault tolerant mode with Zookeeper.
```

Comment:
is this link still needed now that we have a section earlier for HA mode?

Reply:
moved this tips earlier